runner/server/server.go

```diff
@@ -121,14 +124,14 @@ func (s *Server) Start() error {
 
 		// Start http server
 		corsObj := utils.CreateCorsObject()
-		logrus.Infof("Starting server proxy on port %s", s.config.ProxyPort)
+		helpers.Logger.LogInfo(helpers.GetInternalRequestID(), fmt.Sprintf("Starting server proxy on port %s", s.config.ProxyPort), nil)
 		if err := http.ListenAndServe(":"+s.config.ProxyPort, corsObj.Handler(router)); err != nil {
-			logrus.Fatalln("Proxy server failed:", err)
+			helpers.Logger.LogFatal(helpers.GetInternalRequestID(), fmt.Sprintf("Proxy server failed: - %v", err), nil)
 		}
 	}()
 
 	// Start the http server
 	corsObj := utils.CreateCorsObject()
-	logrus.Infof("Starting server on port %s", s.config.Port)
+	helpers.Logger.LogInfo(helpers.GetInternalRequestID(), fmt.Sprintf("Starting server on port %s", s.config.Port), nil)
 	return http.ListenAndServe(":"+s.config.Port, corsObj.Handler(s.router))
```

Comment:
wrap this too with the logger middleware

Reply:
As this logger middlware is common to both the binaries. We can make it as a part of helpers package. What do you say ?

Reply:
For now i have created an extra file with same fucntion copy & paste

Reply:
Works